Dockerfile.cpu

```diff
@@ -31,4 +45,4 @@ WORKDIR /workspace/
 
 RUN ln -s /workspace/vllm/tests && ln -s /workspace/vllm/examples && ln -s /workspace/vllm/benchmarks
 
-CMD ["/bin/bash"]
+ENTRYPOINT ["python3", "-m", "vllm.entrypoints.openai.api_server"]
```

Comment:
I personally think it's ok to keep this as `/bin/bash`, but it also makes sense to keep this consistent with the CUDA dockerfile... 🤔

Reply:
was trying to fix the feature gap on issues: 
https://github.com/vllm-project/vllm/issues/4771